docs/contributing/building/packaging.md

```diff
@@ -10,6 +10,12 @@ You may use `builder/Makefile.packaging` to generate Ubuntu deb packages. It
 will use docker containers to generate appropriate packages, so you don't need
 to install build dependencies in your OS.
 
+Before running these commands make sure your user is added to the docker group:
+
+```
+sudo usermod -aG docker $USER
+```
+
```

Comment:
I assume this instruction is also true for the Fedora packages, no ? We should add it below the introduction then. I think its a good info.

Reply:
Yes, I think so. I've added the same snippet to the Fedora intro.